src/index.js

```diff
@@ -0,0 +1,73 @@
+const flattener = require('flatten-obj')();
+
+// see http://stackoverflow.com/questions/19098797/fastest-way-to-flatten-un-flatten-nested-json-objects
+// http://jsfiddle.net/WSzec/14/
+/* eslint-disable no-use-before-define, no-param-reassign */
+function loop(cur, prop, result, exclusions) {
+  let isEmpty = true;
+
+  for (const p in cur) {
+    if (Object.prototype.hasOwnProperty.call(cur, p)) {
```

Comment:
what is someone would like to use the module but using properties that are down the prototype chain ?